src/main/java/com/brandwatch/internship/fetchandstoretweets/services/FetchAndStoreService.java

```diff
@@ -0,0 +1,67 @@
+package com.brandwatch.internship.fetchandstoretweets.services;
+
+import com.brandwatch.internship.fetchandstoretweets.entities.Mention;
+import com.brandwatch.internship.fetchandstoretweets.entities.Query;
+import com.brandwatch.internship.fetchandstoretweets.repositories.MentionsRepository;
+import com.brandwatch.internship.fetchandstoretweets.repositories.QueryRepository;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.social.twitter.api.SearchResults;
+import org.springframework.social.twitter.api.Tweet;
+import org.springframework.social.twitter.api.Twitter;
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+@Service
+public class FetchAndStoreService {
+
+    private Twitter twitter;
+
+    private MentionsRepository mentionsRepository;
+    private QueryRepository queryRepository;
+
+    private int counter;
+    private String[] searchStrings = {"#maven", "#spring", "#intellij_idea", "#kafka"};
+
+    @Autowired
+    public FetchAndStoreService(Twitter twitter, MentionsRepository mentionsRepository,
+                                QueryRepository queryRepository) {
+
+        this.twitter = twitter;
+        this.mentionsRepository = mentionsRepository;
+        this.queryRepository = queryRepository;
+
+        this.mentionsRepository.deleteAll();
```

Comment:
Do we want to delete all on every restart?

Reply:
I thought that we just need the data when the program is running. That is why I had this counter instead of making it depend on the DB. Of course I will change it now.